src/get.spec.js

```diff
@@ -0,0 +1,66 @@
+import get from './get'
+// The get function shall be tested to ensure different levels (1,2,3) and 
+// kind (array, object) property paths return the designated property. 
+
+describe('Testing suite for "get"-function', () => {
+    describe('Simple object tests', () => {
+        const obj = { a: 1, b: 3 }
+        it('Should return value on corresponing key', () => {
+            expect(get(obj, 'a')).toBe(1)
+        })
+        it('Should return undefined on non existing value', () => {
+            expect(get(obj, 'c')).toBe(undefined)
+        })
+        it('Should return default value if specified path does not exists', () => {
+            expect(get(obj, 'c', null)).not.toBe(undefined)
+            expect(get(obj, 'c', null)).toBe(null)
+            expect(get(obj, 'c', 'not exists')).toBe('not exists')
+        })
+    })
+    describe('Simple array tests', () => {
+        const arrStr = ['a', 'b', 'c']
+        const arrNmbr = [1, 2, 3]
+        it('Should return first element of array on given path (\'[0]\')', () => {
+            expect(get(arrNmbr, '[0]')).toBe(1)
+            expect(get(arrStr, '[1]')).toBe('b')
+        })
+        it('Should return \'undefined\' on non existing element', () => {
+            expect(get(arrStr, '[3]')).toBe(undefined)
+        })
+        it('Should return \'null\' on non existing element with default value specified as \'null\'', () => {
+            expect(get(arrStr, '[3]', null)).toBe(null)
+        })
+    })
+
+    describe('Complex object testing with inner array and object', () => {
```

Comment:
The below testcases look good to me, however the seccond parameter could accept an array too according to the jsdoc. Maybe you can add similar cases with array instead of string? `'nestedObj1.a' -> ['nestedObj1','a']`